adapter-issuer/kustomize/adapter-issuer/base/deployment.yml

```diff
@@ -27,6 +27,7 @@ spec:
           - secretRef:
               name: issuer-env-secret
         image: issuer-adapter:latest
+        imagePullPolicy: IfNotPresent
```

Comment:
Do we need this change ?

Reply:
missing in some projects and it doesn't update images when you have new one. Problem arises only when you use 'latest' image.

Reply:
Make deploy inside wallet-web or inside k8s folder fails ... to pull the image. This seems to fix that issue

Reply:
removed it